README.md

```diff
@@ -48,11 +48,20 @@ First-timers is built with [Probot](https://probot.github.io/).
 
 <p align="center"><img src="/assets/Issue-Done.png"></p>
 
+### Contributors
 
+Thank you to everyone who has helped with this project.
+<!-- Contributors START
+Michael_McCombue michaelmccombie https://twitter.com/michaelbuilds design
+Contributors END -->
+<!-- Contributors table START -->
+| <img src="https://avatars.githubusercontent.com/michaelmccombie?s=100" width="100" alt="Michael McCombue" /><br /><sub>Michael McCombue</sub>](https://twitter.com/michaelbuilds)<br />🎨 |
+| :---: |
+<!-- Contributors table END -->
+This project follows the [all-contributors](https://github.com/kentcdodds/all-contributors) specification.
+
+🎨	Thank you to @michaelmccombie for contributing the pink Hoodie bot. Find him on [Twitter]().
```

Comment:
Add link to twitter account between ()

Reply:
I'm using an Atom package to generate and it has some bugs. So I'm making the fixes manually. Should be fixed now.

Reply:
Hey @agonzalez0515 !
Just noticed that you are using [all-contributors-atom](https://github.com/o-d-i-n/all-contributors-atom). Thanks for reminding me to release the package (I'm the maintainer 😅 )
I forgot to release after merging the [fix](https://github.com/o-d-i-n/all-contributors-atom/pull/10) for this issue  (calls for automation 🤔 ) ...

PS : It feels weird that something I made could be useful to someone 😛

Reply:
@mbad0la Hi! Awesome! The package is super handy!